spec/integration/foods_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'Foods', type: :system, js: true do
+  before :all do
+    @user = User.create(name: 'Test User', email: 'user@mail.com', password: 'password')
+    @banana = Food.create(name: 'Banana', measurement_unit: 'g', price: 0.10, quantity: 100, user: @user)
+    @apple = Food.create(name: 'Apple', measurement_unit: 'units', price: 0.20, quantity: 100, user: @user)
+  end
+
+  before :each do
+    visit new_user_session_path
+    fill_in 'Email', with: @user.email
+    fill_in 'Password', with: 'password'
+    click_button 'Log in'
+
+    visit foods_path
+  end
+
+  it 'shows the food page information' do
+    expect(page).to have_content('Banana')
+    expect(page).to have_content('Apple')
+
+    expect(page).to have_content('$0.1')
+    expect(page).to have_content('$0.2')
+
+    expect(page).to have_content(@banana.measurement_unit)
+    expect(page).to have_content(@apple.measurement_unit)
+  end
+end
```

Comment:
- [ ] Kindly fix you integretion tests they are not passing, amke sure all of your tests pass

![image](https://github.com/Timbar09/RecipePlug/assets/65068771/b75aab5e-dc11-48f0-9c17-da3c682bc7d2)

Reply:
Thank you for the feedback, @andersonlebon. The test works fine if you test each integration spec file independently.